nexus/src/authz/api_resources.rs

```diff
@@ -150,7 +150,7 @@ impl<T: ApiResource + oso::PolarClass + Clone> AuthorizedResource for T {
 /// This object is used for authorization checks on a Fleet by passing it as the
 /// `resource` argument to [`crate::context::OpContext::authorize()`].  You
 /// don't construct a `Fleet` yourself -- use the global [`FLEET`].
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
```

Comment:
How does this get serialized? `{}`?

Reply:
I think that depends on the serializer. For `serde-json`, which is used by steno, I believe the name of the struct is used:

https://serde.rs/json.html